src/ui/DynamicFacetManager/DynamicFacetManager.ts

```diff
@@ -218,7 +217,7 @@ export class DynamicFacetManager extends Component {
     this.respectMaximumExpandedFacetsThreshold();
 
     this.containerElement.appendChild(fragment);
-    this.element.appendChild(this.containerElement);
+    $$(this.element).prepend(this.containerElement);
```

Comment:
It would be good to extract this line into a method `prependContainer` and use it here and on line 128.